Cleanup Code

TheLengendaryWiz opened the pull request:
PR for Cleaning up code. Not much review required , just some comments. Test it on the arduino.

TheLengendaryWiz committed:
Cleanup Code

Tusker13-04 commented:
i am going to keep this PR open will merge it all to base once all phase 1 tasks are complete

TheLengendaryWiz committed:
Added an array and some other things

TheLengendaryWiz committed:
Fixed some small errors

TheLengendaryWiz committed:
Made some changes that probably dont work

TheLengendaryWiz committed:
Cleaned up code (Again)

TheLengendaryWiz committed:
Added py file

legionyt committed:
made a few changes, writing to json and cooldown

updates data to json, and implements cool down. fingers crossed. no testing has been done since i obviously dont have the provisions to do it. i can give fake input but im lazy :)

legionyt committed:
Simple change

legionyt committed:
added data.json

legionyt committed:
umm, made it actually work (not really)

legionyt committed:
:)

Tusker13-04 committed:
few changes

legionyt committed:
cleaned up (slightly)

legionyt committed:
made a registerer

legionyt committed:
ig the final commit of phase 1 :)

Tusker13-04 committed:
Update README.md
